rest-api/src/main/java/com/cloudbees/workflow/flownode/FlowNodeUtil.java

```diff
@@ -249,12 +384,17 @@ public static FlowNode getStageEndNode(FlowNode stageStartNode) {
         return allNodesSorted.get(allNodesSorted.size() - 1);
     }
 
+    @CheckForNull
     /**
      * Get the last node to start in a flow.
      * @param execution The flow execution.
      * @return The last node to start in the flow.
      */
     public static FlowNode getFlowEndNode(FlowExecution execution) {
+        if (execution == null) {
```

Comment:
:ant: again use FindBugs and decide whether you intend to allow a parameter to be null or not.